src/content/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/android-sdk-api-guide.mdx

```diff
@@ -412,6 +412,16 @@ The following table lists [all the New Relic Android SDK API calls](/docs/mobile
         See `recordHandledException()`.
       </td>
     </tr>
+
+    <tr>
+      <td>
+        Record JavaScript errors from ReactNative apps
+      </td>
+
+      <td>
+        See [`recordJSErrorException`](/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/recordjserrorexception-android-agent-api).
```

Comment:
@ywang-nr do we need this URL instead: /docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/record-jserror-exception

Reply:
resolved